index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fruit Samurai</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/67e421d227.js" crossorigin="anonymous"></script>
+    <script src="https://code.jquery.com/jquery-3.3.1.js" crossorigin="anonymous"></script>
+    <audio id="audio">
+        <source src="Fruit Ninja Combo Sound [UI Sounds] (mp3cut.net).mp3" type="audio/mpeg">
+    </audio>
+</head>
+
+<body>
+    <div id="background-img">
+
+        <div id="top-bar">
+            <div id="name">Fruit Samurai</div>
+            <div class="timer"></div>
+        </div>
+
+        <div id="play-area"></div>
+
+        <div id="bottom-bar">
+            <div id="level">Level: 0</div>
```

Comment:
you don't really need all those, because you can just run the render function in the beginning and it will render it